app/src/main/res/values/strings.xml

```diff
@@ -100,6 +102,10 @@
     <string name="please_select_at_least_one">Please select at least one!</string>
     <string name="you_can_t_select_more_than_3">You can not select more than 3!</string>
     <string name="ready_to_go">You are ready to go !</string>
+    <string name="set_password">Enter Password</string>
+    <string name="set_email">Enter Email Address</string>
```

Comment:
And this one too ? I start to think Cirrus is broken, all the strings you added appear unused